src/django_components/templatetags/component_tags.py

```diff
@@ -393,7 +469,3 @@ def _get_positional_param(
 
 def is_wrapped_in_quotes(s: str) -> bool:
     return s.startswith(('"', "'")) and s[0] == s[-1]
-
-
-def strip_quotes(s: str) -> str:
```

Comment:
Instead of stripping quotes outselves, I used `FilterExpression.resolve()` to do that for us here https://github.com/EmilStenstrom/django-components/pull/495/files#diff-7345171ed38e1dd2e4b8199b65a2fd082323bdb331ff0ef0dc0dd3bd43f111aeR345